.github/workflows/ci.yml

```
@@ -55,6 +55,8 @@ jobs:
       - uses: julia-actions/cache@v2
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
+      - name: Run multithreaded tests
+        run: julia --project=. --color=yes --threads=2 test/runtests.jl
```

```suggestion
      - name: Run single-threaded tests
        uses: julia-actions/julia-runtest@v1
      - name: Run multithreaded tests
        uses: julia-actions/julia-runtest@v1
        env:
          JULIA_NUM_THREADS: 2
```
Maybe more canonical?

---

Good idea!